### PR TITLE
Grab payee name for Abanca gocardless integration

### PR DESCRIPTION
**Pull request by daca11**
Current GoCardless integration with Abanca is not able to get Payee name because `creditorName` and `debtorName` are empty.
That information is actually in the `remittanceInformationStructured`. This fix adds that information to `creditorName` and `debtorName` so the payee name is correctly imported into the transaction correctly.

### CODE REVIEW

**Review by psybers — CHANGES_REQUESTED** [review]
Can you add a test case? See https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/banks/tests/belfius_gkccbebb.spec.js for an example.

**Comment by psybers** [comment]
Be sure to update this file too: https://github.com/actualbudget/actual-server/blob/master/src/app-gocardless/tests/bank-factory.spec.js

**Comment by psybers** [comment]
Actually hold off on modifying the test for bank factory. #367 should make that no longer necessary.

**Comment by MatissJanis** [comment]
Thanks for working on this! Would you mind updating the PR to use prop spreading for `Fallback`? Here's an example bank provider: https://github.com/actualbudget/actual-server/blob/aa22e6951dd90fc03e086593f001d0a002b6e324/src/app-gocardless/banks/american-express-aesudef1.js#L7

We recently merged some changes to simplify them.

Happy to merge this PR once that's done + the conflict is solved!

**Comment by daca11** [comment]
Added a test and the ...Fallback line.
Sorry for the trial-and-error commits, as a DevOps I never worked with ts. I hope the test is correct, I just based my work on the existing files 😄

File: src/app-gocardless/banks/abanca-caglesmm.js
Comment (MatissJanis) [inline src/app-gocardless/banks/abanca-caglesmm.js:31]: Can remove these. The prop spreading on line 5 takes care of adding them.

```suggestion
```

File: src/app-gocardless/banks/abanca-caglesmm.js
Comment (MatissJanis) [inline src/app-gocardless/banks/abanca-caglesmm.js:14]:
```suggestion
```

**Review by MatissJanis — CHANGES_REQUESTED** [review]

**Review by MatissJanis — APPROVED** [review]
Thanks!

**Pull request merged**
